cyhy/mailer/cli.py

```diff
@@ -31,6 +31,27 @@
   -c --db-creds-file=FILENAME       A YAML file containing the Cyber
                                     Hygiene database credentials.
                                     [default: /run/secrets/database_creds.yml]
+  -e --csa-email-file=FILENAME      A YAML file associating each state
```

Comment:
Since this file should associate multiple emails I think the argument should use the plural. Note: I spaced and accompanying suggestions are further down.

```suggestion
  -e --csa-emails-file=FILENAME     A YAML file associating each state
```